pkg/cloud/cloud.go

```diff
@@ -53,7 +53,8 @@ func GetCloudConfigTransformer(platformStatus *configv1.PlatformStatus) (cloudCo
 		if azurestack.IsAzureStackHub(platformStatus) {
 			return azurestack.CloudConfigTransformer, true, nil
 		}
-		return nil, true, nil
+		// We need to use the azure CloudConfigTransformer to fix OCPBUGS-25483
```

Comment:
This comment is unlikely to age well, and git blame is probably going to be easier for us to use to track the history of this.

Reply:
Right - so ok to just remove it?

Reply:
Yeah, I expect in the future this transformer will handle fixes for multiple things, not just this one bug, so it will probably be wrong pretty quickly